scss/_patterns_breadcrumbs.scss

```
@@ -25,7 +25,7 @@
     }
 
     &:not(:first-of-type)::before {
-      content: '\203A';
+      content: '\2215';
```

Because this is not any special character I think it can simply be directly written without unicode.

```suggestion
      content: '/';
```